recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "20.2.2" %}
-{% set sha256 = "129cc1b7e6b00c1740c5c6f285a2c6207e18f05cdeb71af8107c3fedf48f23de" %}
+{% set version = "20.4.1" %}
+{% set sha256 = "a7f3333f01037b088dc9701eae2cc46e2cc6a784554df16fc61a1a07c52e87b9" %}
```

Comment:
Okay, their release history is a bit mess but I think we want to be pulling down something more recent https://github.com/m3g/packmol/releases

Reply:
I'm okay with this getting merged - it's still more recent than what's currently online: https://anaconda.org/conda-forge/packmol/files

Reply:
Do you know if there is a demand any particular version on conda-forge? I'm happy to merge this in, then work on getting the latest built

Reply:
No idea